packages/account-v2/src/modules/src/AddressFields/AddressFields.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
some folders in src/modules have index file while some others don't. can we standardize to one approach

Reply:
We are going to follow the approach of creating index files, created missing index files